### ISSUE TITLE
heredoc

### ISSUE THREAD

**Issue by FichteFoll** [jfcherng/Sublime-ShellScriptImproved#35]
```sh
cat <<_ACEOF
\`configure' configures Aegisub aegisub_version to adapt to many kinds of systems.

Usage: $0 [OPTION]... [VAR=VALUE]...

_ACEOF
```

**Comment by jfcherng** [comment]
Thanks. This should be fixed in `1.2.18`.

---

But I still have no idea how to parse the advanced following one.

```bash
cat <<_ACEOF; echo hello;
world
_ACEOF
```
The above script prints 
```
world
hello
```

**Comment by FichteFoll** [comment]
Check out the Ruby syntax. You can get around this by pushing two contexts and have the outer context remove the inner's meta_scope with clear_scopes.
